guides/source/action_view_overview.md

```diff
@@ -517,43 +556,50 @@ CAUTION: Only keyword arguments are supported. Defining positional or block argu
+Layouts can be used to render a common view template around the results of Rails controller actions. A Rails application can have multiple layouts that pages can be rendered within. 
+
+For example, an application might have one layout for a logged in user and another for the marketing part of the site. The logged in user layout might include top-level navigation that should be present across many controller actions. The sales layout for a SaaS app might include top-level navigation for things like "Pricing" and "Contact Us" pages. Different layouts typically have a different header and footer content. 
+
+To find the layout for the current controller action, Rails first looks for a file in `app/views/layouts` with the same base name as the controller. For example, rendering actions from the PhotosController class will `use app/views/layouts/photos.html.erb` (or `app/views/layouts/photos.builder`). 
+
+If such a controller-specific layout does not exist, Rails will use `app/views/layouts/application.html.erb` (or `app/views/layouts/application.builder`). Rails also provides more ways to assign specific layouts to individual controllers and actions. See [Layouts and Rendering guide](layouts_and_rendering.html#finding-layouts) for more.
+
+You can learn more about layouts in general in the [Layouts and Rendering in Rails](layouts_and_rendering.html) guide.
 
 ### Partial Layouts
 
 Partials can have their own layouts applied to them. These layouts are different from those applied to a controller action, but they work in a similar fashion.
 
-Let's say we're displaying an article on a page which should be wrapped in a `div` for display purposes. Firstly, we'll create a new `Article`:
+Let's say we're displaying an article on a page which should be wrapped in a `div` for display purposes. First, we'll create a new `Article`:
 
 ```ruby
 Article.create(body: 'Partial Layouts are cool!')
 ```
 
 In the `show` template, we'll render the `_article` partial wrapped in the `box` layout:
 
-**articles/show.html.erb**
-
-```erb
+```html+erb
+<%# articles/show.html.erb %>
 <%= render partial: 'article', layout: 'box', locals: { article: @article } %>
 ```
 
 The `box` layout simply wraps the `_article` partial in a `div`:
 
-**articles/_box.html.erb**
-
 ```html+erb
+<%# articles/_box.html.erb %>
 <div class='box'>
   <%= yield %>
 </div>
 ```
 
-Note that the partial layout has access to the local `article` variable that was passed into the `render` call. However, unlike application-wide layouts, partial layouts still have the underscore prefix.
+Note that the partial layout has access to the local `article` variable that was passed into the `render` call. Though it is not being used within `_box.html.erb` in this case.
```

Comment:
Note that the partial layout has access to the local `article` variable that was passed into the `render` call, although it is not being used within `_box.html.erb` in this case.